src/services/deleteTimeEntries.ts

```diff
@@ -1,12 +1,13 @@
 export async function deleteTimeEntry(id: number): Promise<[any]> {
-  const response = await fetch(
-    `https://my-json-server.typicode.com/humanoidsbv/team-awesome-amijs/time-entries/${id}`,
-    {
-      method: "DELETE",
-      headers: {
-        "Content-Type": "application/json",
-      },
+  const url = process.env.NEXT_PUBLIC_LOCAL_HOST
+    ? "http://localhost:3006"
+    : "https://my-json-server.typicode.com/humanoidsbv/team-awesome-amijs";
```

Comment:
```suggestion
  const url = process.env.NEXT_PUBLIC_LOCAL_HOST || "https://my-json-server.typicode.com/humanoidsbv/team-awesome-amijs";
```

This way the local_host environment variable is used if **present**, otherwise the string following is used